content/channels/index.textile

```diff
@@ -1647,11 +1681,10 @@ let listener = channel.on { stateChange in
 
 ```[realtime_flutter]
 final stateChangeListener = channel
-  .on()
-  .subscribe((ably.ChannelStateChange stateChange) {
-    print('channel state is ${stateChange.current.toString().split('.').last}');
-  }
-);
+    .on()
+    .listen((ably.ChannelStateChange stateChange) {
+  print('channel state is ${stateChange.current.toString().split('.').last}');
```

Comment:
I think `toString().split('.').last` looks too complicated, since dart 2.15 you can use `name` or we can leave `toString()` without grooming output

```suggestion
  print('channel state is ${stateChange.current.name}');
```

Reply:
Ahh, yes that's much cleaner.